CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.3.4
+
+- CSP: Do not deduplicate alternate schema source expressions (@keithamus): https://github.com/github/secure_headers/pull/478
```

Comment:
This file hasn't been particularly consistent about formatting, but I've gone with Markdown bullet formatting and including the PR link for maximum readability.